Models/AzureFileProvider.cs

```diff
@@ -64,6 +64,16 @@ protected async Task<FileManagerResponse> GetFilesAsync(string path, string filt
             FileManagerDirectoryContent cwd = new FileManagerDirectoryContent();
             try
             {
+                var blobPages = container.GetBlobsAsync(prefix: path).AsPages().GetAsyncEnumerator();
+                await blobPages.MoveNextAsync();
+                bool directoryExists = blobPages.Current.Values.Count() > 0;
+                if (!directoryExists) {
+                    ErrorDetails er = new ErrorDetails();
+                    er.Message = "Could not find a part of the path " + "'" + path + "'" + ".";
```

Comment:
do we have localization on error message?

Reply:
Hi @gsumankumar ,

We don't have location support for this. It's common file not fount related exception we throw in the component level.

Reply:
Customer can modify this code based on their need.